pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "opendrift-leeway"
```

Comment:
[leeway](https://pypi.org/project/leeway/) is already taken on PyPI, what do you think about the name "opendrift-leeway"?

Reply:
Good idea